server/project/project.go

```diff
@@ -405,8 +405,7 @@ func (s *Server) GetSyncWindowsState(ctx context.Context, q *project.SyncWindows
 }
 
 func (s *Server) NormalizeProjs() {
-	ctx := context.Background()
-	projList, err := s.List(ctx, &project.ProjectQuery{})
+	projList, err := s.appclientset.ArgoprojV1alpha1().AppProjects(s.ns).List(metav1.ListOptions{})
 	if err != nil {
 		return
 	}
```

Comment:
I noticed another issue here. If confict error happens that we need to reload project and try to normalize it again. I I think we should move retry whole `NormalizeProjs` of update fails due to conflict error so it reload more recent project and try to normalize them again.